### PR TITLE
Fix no filter condition bug by pagination

### PR DESCRIPTION
**Pull request by tienhoah**

### CODE REVIEW

**Comment by tienhoah** [comment]
[Improve Method dropdown filter on Collecting and Submitted pages to show user it's filtering](https://trello.com/c/ryHxBIWM/977-improve-method-dropdown-filter-on-collecting-and-submitted-pages-to-show-user-its-filtering)

File: src/app/reference/controllers/benthic-attributes.ctrl.js [hunk 1]
```diff
@@ -63,27 +64,64 @@ angular.module('app.reference').controller('BenthicAttributesCtrl', [
             return regions.join(', ') || '-';
           }
         }
-      ]
+      ],
+      toolbar: {
+        template: 'app/reference/partials/fish-families-toolbar.tpl.html',
```
Comment (saanobhaai) [inline src/app/reference/controllers/benthic-attributes.ctrl.js]: `fish-families...` in `benthic-attributes...`?

File: src/app/reference/controllers/benthic-attributes.ctrl.js [hunk 2]
```diff
@@ -63,27 +64,64 @@ angular.module('app.reference').controller('BenthicAttributesCtrl', [
...
+        'benthicAttributeRecordsCount'
+      );
     });
 
+    $scope.tableControl.getFilteredRecordsCount = function() {
+      const tableRecordsTotal =
+        $scope.tableControl.getPaginationTable() &&
+        $scope.tableControl.getPaginationTable().total;
+
+      return `${tableRecordsTotal}/${benthicAttributeRecordsCount}`;
```
Comment (saanobhaai) [inline src/app/reference/controllers/benthic-attributes.ctrl.js]: We should protect against dividing by 0 here

Comment (tienhoah) [reply src/app/reference/controllers/benthic-attributes.ctrl.js]: it's inside string literals, not dividing by 0

File: src/app/reference/controllers/benthic-attributes.ctrl.js (hunk 1 above)
Comment (tienhoah) [reply src/app/reference/controllers/benthic-attributes.ctrl.js]: my mistake while copy/paste => changed

**Pull request merged**